src/Array2D.luna

```diff
@@ -0,0 +1,235 @@
+import Std.Foreign.C.Value
+import Std.Foreign
+
+import Dataframes.Utils
+import Dataframes.Types
+
+class Size:
```

Comment:
I would call it `Shape` as a nod towards `numpy` and `Matlab` users coming to Luna ;)